api-specs/Gateway-EE/3.9/kong-ee.yaml

```diff
@@ -16158,6 +16230,60 @@ paths:
       tags:
       - cache
 
+  '/config':
+    post:
+      summary: Apply Declarative Config
+      tags:
+        - config
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+      requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+            application/yaml:
+              schema:
+                type: object
+            multipart/form-data:
+              schema:
+                type: object
+                properties:
+                  config:
+                    type: string
+                    format: binary
+                    example: /path/to/<config-file>
+                    description: Configuration file in JSON or YAML.
```

Comment:
```suggestion
                    example: /path/to/<config-file>
                    description: Configuration file in JSON or YAML.
```

This doesn't render.

Reply:
![image](https://github.com/user-attachments/assets/e874afe3-d959-4d30-bb5f-de76b271fb56)

Reply:
The description renders if you select multipart/form-data, but not sure why the example doesn't

Reply:
Is multipart correct here? Its not just uploading the yaml configuration? 

For the yaml config you can make it render by changing it like this: 

```
application/yaml:
              schema:
                type: object
              example: |
                services:
                  - name: example-service
                    url: http://example.com
                    routes:
                      - name: example-route
                        paths:
                          - /example
```

Which would render like this: 
![image](https://github.com/user-attachments/assets/27807748-7989-4be0-9e08-c179e9ce6c04)

Reply:
Yes miltipart/form-data and application/yaml are both supported, I tested both